backend/inspirehep/records/api/literature.py

```diff
@@ -133,6 +135,11 @@ def create(
         else:
             push_to_orcid(record)
             push_to_hal(record)
+        if (
+            not disable_disambiguation
+            and current_app.config["FEATURE_FLAG_ENABLE_AUTHOR_DISAMBIGUATION"]
+        ):
+            disambiguate_authors.delay(str(record.id))
```

Comment:
not sure if it will work, the record is not created yet

Reply:
so here you have nested session: https://github.com/inspirehep/inspirehep/blob/2980d09ea36b03613323eb668eca9baadc83b8d9/backend/inspirehep/records/api/literature.py#L121-L125
and here from `InspireRecord` you add the record to the db session https://github.com/inspirehep/inspirehep/blob/2980d09ea36b03613323eb668eca9baadc83b8d9/backend/inspirehep/records/api/base.py#L272